src/server.js

```diff
@@ -208,4 +210,44 @@ app.get('/api/lessons/:lessonId/stats', privateRoutes, async (req, res) => {
     }
 });
 
+const fetchMobile = async (id) => {
+    const number = await users.getUserMobileById(id);
+    return number[0];
+};
+
+const sendMessageToStudent = async (className, mobileNumber, res) => {
+    contactClient.messages
+        .create({
+            body: `You missed today's ${className.name} lesson, please make sure you catch up on content and attend the next one! If you are not able to attend for any reason please let me know!`,
+            from: twilioMobileNumebr,
```

Comment:
💅 To reflect the change above:

```suggestion
            from: twilioMobileNumber,
```